bots/task/github.py

```diff
@@ -83,6 +82,25 @@ def write(self, value):
         with open(self.path, 'a') as f:
             f.write(value)
 
+class GitHubError(RuntimeError):
+    """Raise when getting an error from the GitHub API
+
+    We used to raise `RuntimeError` before. Subclass from that, so that client
+    code depending on it continues to work.
+    """
+
+    def __init__(self, url, response):
+        self.url = url
+        self.data = response.get('data')
+        self.status = response.get('status')
```

Comment:
Does this contain the token? I think you had a situation locally that would throw this error, right? Can you please make double-sure that it isn't in status or data, or at least gets filtered out here?

Reply:
The token only gets passed into request headers. As far as I know, GitHub doesn't return it in *any* field of the response. It's definitely not in the `url`, `data` (the error object), or `status` (the http status code).

Reply:
Ack, thanks!